src/measure_theory/borel_space.lean

```diff
@@ -383,6 +414,11 @@ lemma measurable.mul [has_mul α] [has_continuous_mul α] [second_countable_topo
   {f : δ → α} {g : δ → α} : measurable f → measurable g → measurable (λa, f a * g a) :=
 continuous_mul.measurable2
 
+@[to_additive]
+lemma measurable.mul' [has_mul α] [has_continuous_mul α] [second_countable_topology α]
+  {f : δ → α} {g : δ → α} : measurable f → measurable g → measurable (f * g) :=
+measurable.mul
```

Comment:
What's the difference between `measurable.mul` and `measurable.mul'`?

Reply:
I see: `f * g` vs `λ a, f a * g a`.

Reply:
Correct. When using `rw` to rewrite integrals and want to use this lemmas to prove an argument of your rewrite rule, it's very important that the map matches exactly.